content/wasm-languages/assemblyscript.md

```diff
@@ -53,32 +53,32 @@ This section provides a basic example of building AssemblyScript from source.
 
 Set up the project like this:
 
-```console
+```bash
 $ npm init
 # Answer questions
-$ npm install --save @assemblyscript/loader
+```
 
-added 1 package, and audited 2 packages in 986ms
+Install the TS compiler:
```

Comment:
```suggestion
Install the ActionScript compiler:
```

Reply:
ActionScript or AssemblyScript?

Reply:
Fixed, thanks.  I was having Flash/SWF flashbacks, I guess.

Reply:
Thanks @dicej and @itowlson 
The latest commit has these changes in.